### PR TITLE
chore: migrate to mm webview npm registry

### PR DESCRIPTION
**Pull request by tommasini**
## **Description**

This PR migrates `react-native-webview` to `@metamask/react-native-webview` on the webview upgrade branch branch

This is the [PR](https://github.com/MetaMask/metamask-mobile/pull/7759) of the main webview upgrade branch.

* Removed react native animated fox package and created the component of react native animated fox.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by legobeat** [comment]
Failing test addressed [here](https://github.com/MetaMask/react-native-animated-fox/pull/5#issuecomment-2121418420).

**Review by brianacnguyen — APPROVED** [review]

File: app/components/Base/AnimatedFox.js [hunk 1]
```diff
@@ -0,0 +1,1574 @@
...
+    this.subscription.unsubscribe();
+  }
+
+  webview = React.createRef();
+
+  render() {
+    return (
+      <WebView
+        ref={this.webview}
+        style={{ flex: 1 }}
```
Comment (legobeat) [inline app/components/Base/AnimatedFox.js:60]: [This is failing lint](https://github.com/MetaMask/metamask-mobile/actions/runs/9200230452/job/25306515600?pr=9695):

```
 /home/runner/work/metamask-mobile/metamask-mobile/app/components/Base/AnimatedFox.js
Error:   60:16  error  Inline style: { flex: 1 }  react-native/no-inline-styles
```

**Review by legobeat — DISMISSED** [review]

File: app/components/Base/AnimatedFox.js (hunk 1 above)
Comment (legobeat) [inline app/components/Base/AnimatedFox.js:60]:
```suggestion
        // eslint-disable-next-line react-native/no-inline-styles
        style={{ flex: 1 }}
```
I guess the right thing to do is to add a proper style declaration, though.

Comment (tommasini) [reply app/components/Base/AnimatedFox.js]: Added a new file with the component all refactored: https://github.com/MetaMask/metamask-mobile/pull/9695/commits/070ebc43e366f9633398b7c88fa2257dfb2c3aa7

**Review by sethkfman — APPROVED** [review]
LGTM

**Comment by legobeat** [comment]
> LGTM

@sethkfman A lot of failing Android E2E tests, though...? https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/895704f8-84bc-4e71-afd1-3348764f2211

**Comment by tommasini** [comment]
They are failing on the main update branch as well, the last e2e smoke passing was April third: https://github.com/MetaMask/metamask-mobile/pull/7759/commits

So there is 12 commits to review

**Comment by jpcloureiro** [comment]
We'll continue with the E2E work in #7759 as discussed with @tommasini today

cc @cortisiko @legobeat

**Pull request merged**